packages/react-charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -7,7 +7,7 @@ import { FocusZone, FocusZoneDirection, FocusZoneTabbableElements } from '@fluen
 import { IAccessibilityProps, ChartHoverCard, ILegend, Legends } from '../../index';
 import { Pie } from './Pie/index';
 import { IChartDataPoint, IChartProps, IDonutChartProps, IDonutChartStyleProps, IDonutChartStyles } from './index';
-import { convertToLocaleString, getAccessibleDataObject } from '../../utilities/index';
+import { getAccessibleDataObject } from '../../utilities/index';
```

Comment:
Can you add examples for this change.

Reply:
Also add tests corresponding to these props.

Reply:
done